bedrock/firefox/views.py

```diff
@@ -387,22 +387,14 @@ def get_context_data(self, **kwargs):
         return ctx
 
     def get_template_names(self):
-        funnelcake = self.request.GET.get('f', '')
         locale = l10n_utils.get_locale(self.request)
         version = self.kwargs.get('version') or ''
 
         if detect_channel(version) == 'alpha':
             template = 'firefox/dev-firstrun.html'
         elif show_40_firstrun(version):
-            # New onboarding animation funnecake
-            if locale == 'en-US' and funnelcake in ['99', '100', '111', '112', '113']:
-                template = 'firefox/firstrun/new-firstrun.html'
-            # Yahoo search retention funnelcake test
-            elif locale == 'en-US' and funnelcake == '92':
-                template = 'firefox/firstrun/yahoo-retention.html'
-            elif locale == 'en-US' and funnelcake == '90':
-                # ravioli/katie couric promo
-                template = 'firefox/firstrun/ravioli.html'
+            if lang_file_is_active('firefox/firstrun/onboarding', locale):
```

Comment:
@flodolo new template here that will require translating. Happy to rename the .lang file to anything you prefer.